lms/djangoapps/certificates/admin.py

```diff
@@ -64,7 +64,7 @@ class CertificateGenerationCourseSettingAdmin(admin.ModelAdmin):
     """
     Django admin customizations for CertificateGenerationCourseSetting model
     """
-    list_display = ('course_key', 'enabled')
+    list_display = ('course_key', 'self_generation_enabled', 'language_specific_templates_enabled')
     search_fields = ('course_key',)
```

Comment:
I added a comment to the other thread regarding the exclusion of `enabled`: 
> If you do not exclude it, does that mean it will be available in the form for people to set? If so, I think it would be safer to just exclude it as it reduces the likelihood that someone might be saving to that column while our migration to copy the data to the new field is running. 
@clintonb, @feanil, thoughts?